buildprocess/configureWebpack.js

```diff
@@ -303,12 +152,46 @@ function configureWebpack(
     }
   };
 
-  config.plugins = (config.plugins || []).concat([
-    new StringReplacePlugin(),
-    new webpack.IgnorePlugin(/^\.\/locale$/, /moment$/)
-  ]);
+  config.plugins = config.plugins || [];
 
-  // Fork type checking to a separate thread
+  // Do not import momentjs locale files
+  // Saves ~500kb (unzipped)
+  config.plugins.push(
+    new webpack.IgnorePlugin({
+      resourceRegExp: /^\.\/locale$/,
+      contextRegExp: /moment$/
+    })
+  );
+
+  // Copy assimpjs.wasm file - this is used by AssImpCatalogItem (see AssImpCatalogItem.forceLoadMapItems())
+  config.plugins.push(
+    new CopyPlugin({
+      patterns: [
+        {
+          from: require.resolve("assimpjs/dist/assimpjs.wasm"),
+          to: "assimpjs.wasm"
+        }
+      ]
+    })
+  );
+
+  // Handle reference to Buffer in geojson-merge
+  config.plugins.push(
+    new webpack.ProvidePlugin({
+      Buffer: ["buffer", "Buffer"]
+    })
+  );
+
+  // Define NODE_ENV from mode setting - used in a few places in Terria
+  config.plugins.push(
+    new webpack.DefinePlugin({
+      "process.env.NODE_ENV": JSON.stringify(
+        devMode ? "development" : "production"
+      )
+    })
+  );
+
+  // Fork TypeScript type checking to a separate thread
```

Comment:
I can see the fork-ts-checker-webpack-plugin 1.0 had support for webpack 5 alpha back in 2019 according to the release notes, but I'm guessing things have changed in webpack 5 since that time. Should the `fork-ts-checker-webpack-plugin` and `fork-ts-checker-notifier-webpack-plugin` be bumped to the latest version at the same time as upgrading to webpack 5?

Reply:
Running `yarn gulp build` with empty `wwwroot/build` runtimes:

Webpack 4: 22.3s
Webpack 5 (this branch): 43.8s
Webpack 5 (this branch) + fork-ts-* version 9.x: 21s

so I suggest bumping the versions even if it isn't strictly required.